docs/tutorial/database.rst

```diff
@@ -127,6 +128,7 @@ Add the Python functions that will run these SQL commands to the
 
 
     @click.command('init-db')
+    @with_appcontext
```

Comment:
i don't know why this is needed. didn't get that far yet, but this should unblock newcomers like me.